packages/frontend-plugin-api/src/wiring/createExtensionBlueprint.ts

```diff
@@ -522,7 +529,7 @@ export function createExtensionBlueprint<
   } as ExtensionBlueprint<{
     kind: TKind;
     params: TParams;
-    output: UOutput;
+    output: any;
```

Comment:
Could this have been `ExtensionDataRef[]` and remove the `as ExtensionDataRef[]` above?